LegitimateURLShortener.txt

```diff
@@ -1847,6 +1847,14 @@ $removeparam=iv_,domain=allbirds.com
 ||weidian.com^$removeparam=ifr
 ||weidian.com^$removeparam=share_relation
 
+! https://www.etsy.com/listing/655283263/docking-stationgift-for-men-for?organic_search_click=1&pop=1
+! https://www.etsy.com/listing/984664013/personalized-leather-walletmens?plkey=e61127eaa78c1a3d517e415d3e2014db7e43e6f5%3A984664013&pro=1&frs=1
+$removeparam=organic_search_click,domain=etsy.com
+$removeparam=pop,domain=etsy.com
```

Comment:
This seems like it could break something.

Reply:
I haven't noticed any breakage and someone can send you link without those parameters anyway so i'd assume that the site should work fine without those.

Reply:
Ok. Thanks!